homeassistant/components/lifx/light.py

```diff
@@ -725,11 +727,17 @@ class LIFXColor(LIFXLight):
     """Representation of a color LIFX light."""
 
     @property
-    def supported_features(self):
-        """Flag supported features."""
-        support = super().supported_features
-        support |= SUPPORT_COLOR
-        return support
+    def color_mode(self) -> str:
+        """Return the color mode of the light."""
+        sat = self.bulb.color[1]
+        if sat:
+            return ColorMode.HS
+        return ColorMode.COLOR_TEMP
+
+    @property
+    def supported_color_modes(self) -> set[str] | None:
```

Comment:
```suggestion
    def supported_color_modes(self) -> set[ColorMode]:
```

Reply:
This recommendation might need https://github.com/home-assistant/core/pull/70806